modules/apps/dynamic-data-mapping/dynamic-data-mapping-form-renderer/src/main/java/com/liferay/dynamic/data/mapping/form/renderer/internal/servlet/taglib/DDMFormRendererBottomDynamicInclude.java

```diff
@@ -38,11 +37,17 @@ public void include(
 			HttpServletResponse httpServletResponse, String key)
 		throws IOException {
 
-		if (DDMFormFieldTypesThreadLocal.isFieldTypesRequested()) {
+		Object fieldTypesRequested = httpServletRequest.getAttribute(
+			DDMFormFieldTypesDynamicInclude.
+				FIELD_TYPES_REQUESTED_ATTRIBUTE_NAME);
+
+		if (fieldTypesRequested != null) {
```

Comment:
@cleydyr or @rodrigopaulino, wouldn't be necessary check if the value of this object is false?

Reply:
It's a condition for existence. It's true because we have to put a value. The false value doesn't matter.